Doesn't build on Macs with M1

```
make build
go build -ldflags "-X github.com/Staffbase/syseleven-exporter/pkg/version.Version=1.1.2-6-g15da4eb \
		-X github.com/Staffbase/syseleven-exporter/pkg/version.Revision=15da4ebc03e3bd70333b75bfd7bc25ba662977b9 \
		-X github.com/Staffbase/syseleven-exporter/pkg/version.Branch=master \
		-X github.com/Staffbase/syseleven-exporter/pkg/version.BuildUser=mbaur \
		-X github.com/Staffbase/syseleven-exporter/pkg/version.BuildDate=2022-10-12@17:12:10" \
		-o ./bin/syselevenexporter ./cmd/syselevenexporter;
# golang.org/x/sys/unix
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/syscall_darwin.1_13.go:25:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.1_13.go:27:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.1_13.go:40:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:28:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:43:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:59:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:75:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:90:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:105:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:121:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191128015809-6d18c012aee9/unix/zsyscall_darwin_arm64.go:121:3: too many errors
make: *** [build] Error 2
```

--- a/go.mod
+++ b/go.mod
@@ -8,4 +8,5 @@ require (
 	github.com/prometheus/client_golang v1.2.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
+	golang.org/x/sys v0.0.0-20221010170243-090e33056c14 // indirect
 )

